module/TaskManagement/src/TaskManagement/Task.php

```diff
@@ -495,8 +517,9 @@ public function assignShares(array $shares, BasicUser $member) {
 		}, ARRAY_FILTER_USE_BOTH);
 		*/
 
-		if(array_sum($membersShares) != 1) {
-			throw new InvalidArgumentException('The total amount of shares must be 100');
+		$total = round(array_sum($membersShares),3);
+		if($total != 1.0) {
```

Comment:
occhio a fare i confronti in questo modo tra float, non funziona. Se $total è un int viene castato tutto ad intero, allora a quel punto usa 1, non 1.0